_data/simple-icons.json

```
@@ -1433,12 +1433,12 @@
         {
             "title": "BBC",
             "hex": "000000",
-            "source": "https://commons.wikimedia.org/wiki/File:BBC.svg",
+            "source": "https://commons.wikimedia.org/wiki/File:BBC_Logo_2021.svg",
```

```suggestion
            "source": "https://www.bbc.co.uk",
```
SVG is available from the BBC website directly :smile:

---

Updated this and the SVG path in #f2de37863ed724a1be237d7337f60a8d82da722b, but not sure if that’s reflected in this PR now… I’m confused!